x-pack/distributions/internal/observabilitySRE/qa/acceptance/docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+services:
+  logstash:
+    image: docker.elastic.co/logstash/logstash-observability-sre:${OBSERVABILITY_SRE_IMAGE_VERSION:-8.19.0-SNAPSHOT}
+    container_name: fips_test_logstash
+    ports:
+      - "5044:5044"
+    volumes:
+      - ./logstash/config/${LOGSTASH_CONFIG:-logstash-fips.yml}:/usr/share/logstash/config/logstash.yml
+      - ./logstash/pipeline/${LOGSTASH_PIPELINE:-logstash-to-elasticsearch.conf}:/usr/share/logstash/pipeline/logstash.conf
+      - ./certs:/usr/share/logstash/config/certs
+    networks:
+      - elastic
+    depends_on:
+      - elasticsearch
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-fips:${ELASTICSEARCH_IMAGE_VERSION:-8.19.0-SNAPSHOT}
```

Comment:
oops! I forgot to update this to `docker.elastic.co/cloud-release/elasticsearch-cloud-ess-fips`  https://github.com/elastic/logstash/pull/17627

I'll make sure to get that done ASAP and re-run acceptance tests.

Reply:
Done in https://github.com/elastic/logstash/pull/17623/commits/721e13b872aaf1de2e8e5edefd2d57f28be474d1